config/recipes/gclb/01-elastic-stack.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: hulk
+  labels:
+    app: hulk
```

Comment:
Do we need this label?

Reply:
For me personally, it's just convenient to have a single short label that can be used to delete everything belonging to a specific deployment. 
It's also an informal convention used by some tools as well.